db/migrate/20221026132134_bigint_primary_and_foreign_keys.rb

```diff
@@ -60,7 +60,6 @@ class BigintPrimaryAndForeignKeys < ActiveRecord::Migration[7.0]
     CustomStyle => [:id],
     CustomValue => %i[id customized_id custom_field_id],
     Journal::CustomizableJournal => %i[id journal_id custom_field_id],
-    Delayed::Job => [:id],
```

Comment:
I feel this migration will break things in the future, using literals which may or may not exist anymore in the future.
This is of course unrelated to your changes.
And while this may not be the case for delayed job in particular, this might perhaps break later migrations if we just remove entries here in the future. So I don't want to set a precedent.

I suggest keeping the delayed job entry around so we don't have to worry about any other consequences down the line and to set a good example for future changes touching this file. Looking at the code further below it should already work to change the line to:

```
'delayed_jobs' -> [:id], # delayed job removed in favour of good_job see WP #42 or PR #42
```

Reply:
> This is not resolved, is it? The code is still removed. But of course Delayed::Job won't exist anymore. So this should just be done in plain SQL.
> execute_sql "DELETE FROM delayed_jobs WHERE handler LIKE '%job_class: Notifications::CleanupJob%'"

While technically correct, we don't need to remove delayed jobs as the table should be removed. @ba1ash I don't think this is happening currently correct?

Reply:
True, it is not happening right now. I will add it in this PR.